desktop_files/package.json.orig

```diff
@@ -67,7 +67,7 @@
     "react-native-image-crop-picker": "0.18.1",
     "react-native-image-resizer": "1.0.0",
     "react-native-invertible-scroll-view": "1.1.0",
-    "react-native-keychain": "git+https://github.com/status-im/react-native-keychain.git#v.3.0.0-status",
+    "react-native-keychain": "git+https://github.com/status-im/react-native-keychain.git#v.3.0.0-status-1",
```

Comment:
```suggestion
    "react-native-keychain": "git+https://github.com/status-im/react-native-keychain.git#v3.0.0-1-status",
```

The base repo https://github.com/oblador/react-native-keychain/releases uses format `v3.0.0`, so there's no reason for us to do `v.3.0.0`